mmv1/products/netapp/StoragePool.yaml

```diff
@@ -207,3 +208,17 @@ properties:
       Flag indicating that the hot-tier threshold will be auto-increased by 10% of the hot-tier when it hits 100%. Default is true.
       The increment will kick in only if the new size after increment is still less than or equal to storage pool size.
     min_version: 'beta'
+  - name: 'qosType'
+    type: Enum
+    description: |
+      QoS (Quality of Service) type of the storage pool.
+      Possible values are: AUTO, MANUAL.
+    enum_values:
+      - 'QOS_TYPE_UNSPECIFIED'
```

Comment:
does the enum get set by the api? This field isn't covered in the [API reference](https://cloud.google.com/netapp/volumes/docs/reference/rest/v1/projects.locations.storagePools) currently